source/_components/logger.markdown

```diff
@@ -45,11 +45,17 @@ components:
 logger:
   default: critical
   logs:
-    homeassistant.components: info
-    homeassistant.components.rfxtrx: debug
-    homeassistant.components.lifx: warning
-    homeassistant.components.yamaha: debug
+    # log level for HA core
+    homeassistant.core: fatal
+    
+    # log level for SmartThings lights
+    homeassistant.components.smartthings.light: info
```

Comment:
Maybe just `homeassistant.components.smartthings`? Put lights below if needs be. Most likely you'd log a whole integration, not just a small part of it.

Reply:
How about this latest change?  This demonstrates how you can set the log level for an integration (MQTT) and also for a component (SmartThings light).

Reply:
Looks good to me :grin: